gcloud-java-storage/src/main/java/com/google/gcloud/storage/testing/RemoteGcsHelper.java

```diff
@@ -96,14 +96,48 @@ public static String generateBucketName() {
   }
 
   /**
+   * Creates a {@code RemoteGcsHelper} object for the given project id and JSON key input stream.
+   *
+   * @param projectId id of the project to be used for running the tests
+   * @param keyStream input stream for a JSON key
+   * @return A {@code RemoteGcsHelper} object for the provided options.
+   * @throws com.google.gcloud.storage.testing.RemoteGcsHelper.GcsHelperException if
+   *     {@code keyStream} is not a valid JSON key stream
+   */
+  public static RemoteGcsHelper create(String projectId, InputStream keyStream)
```

Comment:
Yes I think we could remove the option and nobody would complain. However, Options might be extended to allow users to customize retryParams and timeouts. 
I don't have a strong opinion, I am happy either way.